src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -362,6 +363,19 @@ extern "C" {
 #define STATUS_ROLLING_BUFFER_NOT_IN_RANGE STATUS_ROLLING_BUFFER_BASE + 0x00000001
 /*!@} */
 
+/////////////////////////////////////////////////////
+/// Exponential Backoff related status codes
+/////////////////////////////////////////////////////
+
+/*! \addtogroup Exponential back off retries status codes
+ * Values are derived from STATUS_EXPONENTIAL_BACKOFF_BASE
+ *  @{
+ */
+#define STATUS_EXPONENTIAL_BACKOFF_BASE                STATUS_ROLLING_BUFFER_BASE + 0x01000000
```

Comment:
The base should always be based off `STATUS_WEBRTC_BASE`. Need to modify this accordingly.

Reply:
Sounds good. We're anyways discarding this PR in favor of the PIC one.